harmonica/tests/test_point_mass.py

```diff
@@ -91,6 +170,119 @@ def test_g_z_symmetry():
     npt.assert_allclose(results[0], -results[1])
 
 
+@pytest.mark.use_numba
+def test_g_z_signal():
```

Comment:
Shouldn't this be called `test_g_z_sign` instead of `test_g_z_signal`?
```suggestion
def test_g_z_sign():
```